### PR TITLE
fix(a380x/flightmodel): adjust climb performance

### LINKED ISSUE #9042: [A380X][A32NX] Performance tables do not match actual engine model
### Aircraft Version

Development

### Build info

```json
master
```

### Describe the bug

The engine model in the cfg files do not match the performance tables (`EngineModel.ts`) used for predictions. 

Required changes: 

- Make EngineModel aircraft-specific
- - Update values for A32NX to reflect engines.cfg values
- Update values for A380X to reflect engines.cfg values

### Expected behavior

The tables should match.

### Steps to reproduce

See description

### References (optional)

_No response_

### Additional info (optional)

_No response_

### Discord Username (optional)

_No response_

### PR DESCRIPTION
**Pull request by BlueberryKing**
Fixes #9042

## Summary of Changes

Tune engine thrust and drag model to be more accurate.

- [x] Adjust performance in .cfg files
- [x] Update engine performance tables for vertical predictions
- [x] Update performance tables for flight model

## Additional context

Discord username (if different from GitHub):

## Testing instructions

- Perform a full flight, ensure you get (close to) accurate predictions throughout the flight.
- Ensure that the climb, cruise, and descent performance is believable.

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

**Comment by chinjh93** [comment]
Tested some  of the values you've amended the plane feels a bit sluggish at climb now. Gross weight of about 373tonnes and my climb was only 2000fpm during the climb from 10000feet to 15000feet with IAS 290knots on open climb. This value was more believable if the gross weight was around 500tonnes or more.

**Comment by BlueberryKing** [comment]
> Tested some of the values you've amended the plane feels a bit sluggish at climb now. Gross weight of about 373tonnes and my climb was only 2000fpm during the climb from 10000feet to 15000feet with IAS 290knots on open climb. This value was more believable if the gross weight was around 500tonnes or more.

Thanks for testing! I noticed the same thing indeed, this was do to an error in my tuning method. I will continue to work on it.

**Comment by 2hwk** [comment]
@BlueberryKing Let me know when you are done so we can do a formal code review


Also changelog needed.

**Comment by chinjh93** [comment]
Has the parameters been tested and merged with the supposed [Another taxi thrust adjustment](https://github.com/flybywiresim/aircraft/pull/9447) Cause i seem to see both pulls involve the same cfg.

**Comment by BlueberryKing** [comment]
> Has the parameters been tested and merged with the supposed [Another taxi thrust adjustment](https://github.com/flybywiresim/aircraft/pull/9447) Cause i seem to see both pulls involve the same cfg.

My changes are now on top of the PR you mention, so they shouldn't conflict.

File: large-files
Comment (Saschl) [inline large-files:1]: Not sure if that change is intended?

**Comment by donstim** [comment]
One test here and it looked very good to me. Although I don't have any data to base it on (other than comparison to the A32NX), the climb rates looked very believable. For my flight, FMS predicted FL 400 at REC MAX, and I got to FL 39.6 before the climb rate stayed below 300 fpm (one definition of REC MAX).

File: large-files
Comment (flogross89) [reply large-files:1]: Doesn't sound like it's intended, better leave it at the same commit as master

**Review by flogross89 — APPROVED** [review]
Very cool, thank you!

**Comment by BlueberryKing** [comment]
Thank you for the reviews! I will address your comments as soon as I can.

**Comment by DJSean00** [comment]
hmm flying this all the textures are pink?

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -25,22 +25,25 @@
 1. [A380X/SD] Added correct ECP ALL button SD page cycling to the A380X - @frankkopp (Frank Kopp)
 1. [A380X/SURV] Fixed BTV setup from F/O side, and ROW issues at G/A with early THR RED - @flogross89 (floridude)
 1. [A380/MFD] Limit keyboard inputs to keys present in the KCCU and remap comma to decimal dot - @beheh (Benedict Etzel)
```
Comment (2hwk) [inline .github/CHANGELOG.md:27]:
```suggestion
1. [A380X/MFD] Limit keyboard inputs to keys present in the KCCU and remap comma to decimal dot - @beheh (Benedict Etzel)
```

Might as well fix this one while you're at it.

**Review by 2hwk — CHANGES_REQUESTED** [review]

File: fbw-a380x/src/systems/shared/src/OperatingAltitudes.tsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ export class A380AltitudeUtils {
    */
   static calculateRecommendedMaxAltitude(gw: number, disa: number, round = 500) {
     return MathUtils.round(
-      54344 -
+      52442 -
```
Comment (2hwk) [inline fbw-a380x/src/systems/shared/src/OperatingAltitudes.tsx:13]: This is intentional correct?

Comment (BlueberryKing) [reply fbw-a380x/src/systems/shared/src/OperatingAltitudes.tsx:13]: Indeed, the REC MAX is updated inline with the new performance numbers.

**Comment by alexr4339** [comment]
Quality Assurance Tester

Discord Username  : alexdnzdmb
Object of testing : #9358
Aircraft          : A380X
Tier of Testing   : 1
Date              : 15/11/2024

Testing Process: 
Full flight EDDS-EDDP, departure only at KLAX. Compared predicitions to actual altitudes

Notes:
Departed EDDS, having around 3500 fpm at 250KIAS and 360t GW of weight, seems believable 

My alt prediciton at TO to DS035 was FL100, passed it at around 9500

NOTGA was predicted at FL153 when at DS035 (12NM distance from DS035 to NOTGA), passed NOTGA at FL149

OKIBA was predicted at NOTGA with FL224, distance is 17NM from NOTGA to OKIBA, actual FL at OKIBA was FL216

RIMKI to LOHRE, 20NM, RIMKI FL298, Pred LOHRE FL337, passing at 334, 356t

Conclusion: FL is predicted around 10% too high (angle of climb 10% too high)

Generel climb perf
After departure CONF0 I was climbing at around 3500 fpm, 250KIAS, 360t, Tout around 5 degC
At FL180 I was climbing at 290 IAS and 2700 fpm, seems believable for 360t
At FL265 1700fpm and 358t, ISA+1, Speed either 290KIAS or Mach 0.81
FL330 1500fpm, ISA-1, Speed either 290KIAS or Mach 0.81


KLAX to OMDB, FOB 204100kg, predicted FOB at dest -1,2T - check whether predicitons mismatch or FOB (calculated by simbrief) is too low

Departure KLAX
190KIAS, 1700feet, FLEX38, 535T, 11degC, 2500fpm, CONF1
250KIAS, 5000feet, CLB, 535T, 5degC, 2100fpm, CONF0

feels a lot more believable

Testing Results: 
Not Passed, needs minor tweaking but already a great improvement! 

Negatives:       
Much better than before, but predictions seem 10% too high

**Comment by Ueeew** [comment]
Made a test flight
Take off weight: 517t
Climb performance from 0-24000 ft feels good: about 1700-2500 fpm
climb rate from 25000-28000 ft is a bit slow (in my opinion): 800-1500 fpm
Climb performance from 29000-33000 ft way too low, 300-500 fpm
Overall: in the first half of the climb to 25000 ft it feels good, compared to the real world Singapor Airlines flight. (same winds, and probably same weight).
In the second half of the climb it feels too slow and the engines are a little too underpowered (compared to the real world SIA flight).referance Flightradar 24
For reference, I left Frankfurt(with real weather and time on)  at the same time as the SIA flight did IRL The real flight needed 31 min for the climb up to 33000 feet and I in the sim needed 43 min for the same climb. 
Feeling good already, but need a bit more tuning at the end of the climb.

**Comment by donstim** [comment]
> @BlueberryKing Do you think we should merge now and correct the higher alt performance with a separate PR or dial down the high-mach changes a bit in this PR?

Seems like the high Mach changes need to be dialed up a bit rather than down. That is, there seems to still be either too little thrust or too much drag at high Mach. I was trying to fly some MMO conditions at a very light weight at FL390 and it took a very long time to achieve that speed.

**Comment by BlueberryKing** [comment]
> @BlueberryKing Do you think we should merge now and correct the higher alt performance with a separate PR or dial down the high-mach changes a bit in this PR?

I am still working on it, but I hope it will be ready to merge soon even if it's not perfect yet.

**Comment by mattking247** [comment]
Quality Assurance Tester Trainee Report

Discord Username : Matt
Object of testing : https://github.com/flybywiresim/aircraft/pull/9358
Aircraft : A380X
Tier of Testing : 1
Date : 26/11/2024

Testing Process:
Full flight OMDB - EGLL, full departure from OMDB Rwy 30R with calm weather. 

Notes:
456.9 tonnes at take off. 

Taxi: aircraft accelerated slowly to 17kts and then stayed there under idle power. during tight corners, the aircraft generally required some small amount of power to continue. Much more realistic. 

Initial climb: 2800fpm THR 88% N1 84% with flaps 1.

Climb: 
250kts at 10,000ft, 2500FPM
290kts at 11500ft, 2200FPM
290kts at 20000ft, 1900FPM
290kts at 25000ft, 1500FPM - THR 97%, N1 91%
290kts at 30000ft, 900FPM - THR 99%, N1 91%
290kts at 32000ft, 600FPM - THR 99%, N1 91.5%
290kts at 33000ft, 500FPM - As above
290kts at 34000ft, 500FPM - As above
290kts at 35000ft, 500FPM - As above
Initial cruise at FL 360.

Climb out seemed appropriate for the weight. 900-500 FPM minute climb rate seemed a little low in the low 30,000s. 


Step Climb - weight 421 tonnes. FL360-380
M.85 at 37000ft, 450FPM - THR 99%, N1 91.5% 
Climbed between FL360 and FL 380 at between 400 and 500FPM. 

Overall - the feel of the aircraft in this configuration seems much better and more appropriately powered compared to previous versions. Slightly underpowered in the low 30,000 ft, but feels significantly better. 

Testing Results:
Not Passed, as I think upper air needs more tweaking. However, much better,

**Comment by BlueberryKing** [comment]
> Quality Assurance Tester Trainee Report
> 
> Discord Username : Matt
> Object of testing : https://github.com/flybywiresim/aircraft/pull/9358
> Aircraft : A380X
> Tier of Testing : 1
> Date : 26/11/2024
> 
> Testing Process:
> Full flight OMDB - EGLL, full departure from OMDB Rwy 30R with calm weather. 
> 
> Notes:
> 456.9 tonnes at take off. 
> 
> Taxi: aircraft accelerated slowly to 17kts and then stayed there under idle power. during tight corners, the aircraft generally required some small amount of power to continue. Much more realistic. 
> 
> Initial climb: 2800fpm THR 88% N1 84% with flaps 1.
> 
> Climb: 
> 250kts at 10,000ft, 2500FPM
> 290kts at 11500ft, 2200FPM
> 290kts at 20000ft, 1900FPM
> 290kts at 25000ft, 1500FPM - THR 97%, N1 91%
> 290kts at 30000ft, 900FPM - THR 99%, N1 91%
> 290kts at 32000ft, 600FPM - THR 99%, N1 91.5%
> 290kts at 33000ft, 500FPM - As above
> 290kts at 34000ft, 500FPM - As above
> 290kts at 35000ft, 500FPM - As above
> Initial cruise at FL 360.
> 
> Climb out seemed appropriate for the weight. 900-500 FPM minute climb rate seemed a little low in the low 30,000s. 
> 
> 
> Step Climb - weight 421 tonnes. FL360-380
> M.85 at 37000ft, 450FPM - THR 99%, N1 91.5% 
> Climbed between FL360 and FL 380 at between 400 and 500FPM. 
> 
> Overall - the feel of the aircraft in this configuration seems much better and more appropriately powered compared to previous versions. Slightly underpowered in the low 30,000 ft, but feels significantly better. 
> 
> Testing Results:
> Not Passed, as I think upper air needs more tweaking. However, much better, 
> 
> **EDIT** Since doing this test, I spoke to a colleague of mine on the A380 fleet. He said that a step climb between FL 360 and FL 380 would normally be around 1000-1200FPM in climb rate. Maybe you would see 600FPM when climbing up to FL400. However, the lower altitude climb rates I mentioned above were pretty accurate according to him. Looks like the climb rates in the FL300-360 was the area that was a little low. 
> 
> 
> 

Thank you for testing! I did also notice a slight deficiency in thrust around FL300, will see if I can adjust a bit further. For the high altitude step climb, I think the issue is that the N1 is slightly too low. I will also look into that.

**Comment by donstim** [comment]
Climb rate will depend on weight and OAT.

**Comment by mattking247** [comment]
Quality Assurance Tester Trainee Report

Discord Username : Matt
Object of testing : https://github.com/flybywiresim/aircraft/pull/9358
Aircraft : A380X
Tier of Testing : 1
Date : 27/11/2024

Testing Process: Previous test copied with similar weights. Then a worst case max take off weight was performed to compare. Results are below.

Both Tests: LFBO departure, runway 14R. Live weather, 4 degrees. Both tests to FL 360, flown on a direct heading without any turns. Tests flown one after the other, so weather was as close to similar as possible. 

Test 1 456.9 tonnes at take off (previous test repeat).

Test 1 - repeat of previous test, altitudes and climb numbers below for reference:
Flap 1 departure, initial climb until accel alt: 2700FPM

250kts  -  4,000ft - 2500FPM - THR 94.3 - N1 85.1
"           -  6,000ft - 2800FPM - THR 95 - N1 85.8
"           -  8,000ft - 2700FPM - THR 95 - N1 86.8
"          - 10,000ft - 2600FPM - THR 94.9 - N1 87.5
290kts - 12,000ft - 2300FPM - THR 93.6 - N1 87.9
"          - 14,000ft - 2300FPM - THR 92.6 - N1 88.4
"          - 16,000ft - 2200FPM - THR 91.7 - N1 88.9
"          - 18,000ft - 2000FPM - THR 91.4 - N1 89.4
"          - 20,000ft - 1800FPM - THR 91.3 - N1 89.9
"          - 22,000ft - 1700FPM - THR 92.9 - N1 90
"          - 24,000ft - 1700FPM - THR 94.5 - N1 90.1
"          - 26,000ft - 1500FPM - THR 99 - N1 89.9
"          - 28,000ft - 1200FPM - THR 99 - N1 89.5
"          - 30,000ft - 900FPM - " - N1 89.4
"          - 32,000ft - 600FPM - " - N1 89.2
"          - 34,000ft - 500FPM - " - N1 89.0
"          - 36,000ft - 500FPM - " - N1 89.0 

Sadly, I did not record the time to FL360 on the previous test I ran. However, for this test it was 26.41. The results are fairly similar to the previous test. 

----------------------------------------------------------------------------

Next, I did a worst case scenario test with all the same test parameters, apart from this time at max take off weight. 
All other variables were the same. Test was conducted immediately following the previous test.

509.4 tonnes at take off.

Test 2 - Max takeoff weight, altitude and climb numbers below for reference: 
Flap 2 departure, initial climb until accel alt: 2600FPM

250kts  -  4,000ft - 2500FPM - THR 93.6 - N1 84
"           -  6,000ft - 2500FPM - THR 95 - N1 85.2
"           -  8,000ft - 2500FPM - THR 95 - N1 86.4
"          - 10,000ft - 2400FPM - THR 94.8 - N1 87.2
290kts - 12,000ft - 2200FPM - THR 93.6 - N1 87.6
"          - 14,000ft - 2200FPM - THR 92.6 - N1 88.2
"          - 16,000ft - 1900FPM - THR 91.7 - N1 88.7
"          - 18,000ft - 1800FPM - THR 91.4 - N1 89.2
"          - 20,000ft - 1800FPM - THR 91.3 - N1 89.6
"          - 22,000ft - 1800FPM - THR 92.8 - N1 89.8
"          - 24,000ft - 1500FPM - THR 94.5 - N1 90.1
"          - 26,000ft - 1400FPM - THR 99 - N1 89.8
"          - 28,000ft - 1300FPM - THR 99 - N1 89.5
"          - 30,000ft - 900FPM - " - N1 89.4
"          - 32,000ft - 600FPM - " - N1 89.2
"          - 34,000ft - 500FPM - " - N1 88.9
"          - 36,000ft - 600FPM - " - N1 89.1

As you can see, the numbers were not significantly different, and the main change in FPM came around the mid to late teens. Time to FL360 at max takeoff weight was 27.35, less than a minute more then the previous test. 

For me, both of these numbers seem relatively accurate throughout the climb, but still seem to washout somewhat more than I would have thought in the 30,000ft area for the non-max TO weight test. For a max take off climb, I think these numbers seem quite accurate. As for the repeat test at 456 tonnes, maybe this lighter weight (relative to max) is where a little tweaking still needs to happen. Interested to hear the thoughts of others though. Clearly, this is all temp and conditions dependent, but at 4 deg C on take off, and clear skies both times, seems a little cooler than standard. 

I am on the fence about a pass/fail. I think this new method should be implemented soon, as it is a lot more accurate than previous versions. I think the max takeoff weight climb performance seems pretty spot on to me. Maybe lighter weights is a little low on FPM climb rate in the 30,000s, but nothing much. If it needs a little tweaking later on, it is much smaller tweaks than the current climb model.

**Comment by Ueeew** [comment]
I have some bug which might help. One comment said under this topic said that the N1 is to low in Climb and Cruise. In my opinion this is true. In fact, the N1 should be higher in climb and cruise than the THR but hat isnt the case. I'm not sure if this has anything to do with the wrong engine model. I just wanted to say this in case it helps.

**Comment by BProduction222** [comment]
If I may add my 2 cents here, and some points to consider before tuning too much on the engines. 

There are 2 major reasons why climb performance is currently exaggerated, especially if you are trying to compare YouTube footage with this rendition in MSFS.

1. Currently the Engines are not being limited by their THR %, as they normally should, but are currently being limited by N1. Which means that you will generally get a higher thrust output in the FBW A380 than you normally would. 
So to fix the first issue, the engines when set to CLB thrust must first follow the THR rating, not N1. You will then already notice a major difference in climb performance. 

2. At my airline, it is common to use Derated Climb thrust for the entire climb phase, which also reduces climb performance. We use D3 for below 520t and D2 for 520t or above. 

Then adding those two variables together, and you get a very good climb performance in the A380.

However if you want more data, I can try and get some footage in a level-d sim next time

**Comment by BlueberryKing** [comment]
> If I may add my 2 cents here, and some points to consider before tuning too much on the engines.
> 
> There are 2 major reasons why climb performance is currently exaggerated, especially if you are trying to compare YouTube footage with this rendition in MSFS, which in my opinion, may not require too much engine tuning.
> 
> 1. Currently the Engines are not being limited by their THR %, as they normally should, but are currently being limited by N1. Which means that you will generally get a higher thrust output in the FBW A380 than you normally would.
>    So to fix the first issue, the engines when set to CLB thrust must first follow the THR rating, not N1. You will then already notice a major difference in climb performance.
> 2. At my airline, it is common to use Derated Climb thrust for the entire climb phase, which also reduces climb performance. We use D3 for below 520t and D2 for 520t or above.
> 
> So if those two things would be properly simulated, you would notice a much more realistic representation of the climb performance of the A380.
> 
> I would recommend the next time you fly the FBW A380, look at the thrust CLB limit when taking off, and adjusting your V/S knob to match your EWD Limiting thrust with your THR %, instead of N1 %.
> 
> However if you want more data, I can try and get some footage in a level-d sim next time

Hi

Thank you for your input. The lack of a derate option does indeed contribute to the problem. That's something that we can hopefully add in the future. Regarding 1., that should only be an issue of the thrust limit. The tuning is done by comparing the thrust output (or climb gradients equivalently) for a given N1 between the sim and a reference. The thrust limit then has to be adjusted such that the correct N1 values are indeed reached during the climb at the given conditions.

And yes, indeed, the thrust limit indication is an N1 limitation rather than a THR% at the moment. This is incorrect. However, as long as it is an N1 limit that's applied to the N1 control loop, that is not much different from calculating an appropriate THR% limit and applying it to the THR% control loop, the resulting N1 is the same.

So in short, we are working on adjusting the thrust output to more realistic levels. The tuning should hopefully not take too much longer. 😄

**Comment by BProduction222** [comment]
> > If I may add my 2 cents here, and some points to consider before tuning too much on the engines.
> > There are 2 major reasons why climb performance is currently exaggerated, especially if you are trying to compare YouTube footage with this rendition in MSFS, which in my opinion, may not require too much engine tuning.
> > 
> > 1. Currently the Engines are not being limited by their THR %, as they normally should, but are currently being limited by N1. Which means that you will generally get a higher thrust output in the FBW A380 than you normally would.
> >    So to fix the first issue, the engines when set to CLB thrust must first follow the THR rating, not N1. You will then already notice a major difference in climb performance.
> > 2. At my airline, it is common to use Derated Climb thrust for the entire climb phase, which also reduces climb performance. We use D3 for below 520t and D2 for 520t or above.
> > 
> > So if those two things would be properly simulated, you would notice a much more realistic representation of the climb performance of the A380.
> > I would recommend the next time you fly the FBW A380, look at the thrust CLB limit when taking off, and adjusting your V/S knob to match your EWD Limiting thrust with your THR %, instead of N1 %.
> > However if you want more data, I can try and get some footage in a level-d sim next time
> 
> Hi
> 
> Thank you for your input. The lack of a derate option does indeed contribute to the problem. That's something that we can hopefully add in the future. Regarding 1., that should only be an issue of the thrust limit. The tuning is done by comparing the thrust output (or climb gradients equivalently) for a given N1 between the sim and a reference. The thrust limit then has to be adjusted such that the correct N1 values are indeed reached during the climb at the given conditions.
> 
> And yes, indeed, the thrust limit indication is an N1 limitation rather than a THR% at the moment. This is incorrect. However, as long as it is an N1 limit that's applied to the N1 control loop, that is not much different from calculating an appropriate THR% limit and applying it to the THR% control loop, the resulting N1 is the same.
> 
> So in short, we are working on adjusting the thrust output to more realistic levels. The tuning should hopefully not take too much longer. 😄


Alright, it looks like you guys have a good method to fix this issue. 
I just wanted to make sure you considered those points, since a lot of people who make videos on it or stream say it is too powerful, but then don't actually limit their thrust correctly and consider derated climbs.

**Comment by G-DRAW** [comment]
Regarding taxi thrusts on the A380, in real life they use around about a maximum of 15% THR and no more that 15% THR, even at the MTOW. So according to real world A380 Pilots, they only use a small amount of thrust to get moving at very heavy weights.

**Comment by donstim** [comment]
> Regarding taxi thrusts on the A380, in real life they use around about a maximum of 15% THR and no more that 15% THR, even at the MTOW. So according to real world A380 Pilots, they only use a small amount of thrust to get moving at very heavy weights.

This PR does not affect taxi thrust. Taxi thrust has already been adjusted with the assistance of IRL A380 pilots on the team. (The THR indication on the A380X is not accurately modeled yet.)

**Comment by BlueberryKing** [comment]
I have made some further adjustments to the thrust output and thrust limits. The thrust output might be a bit too high at high altitudes now, but I prefer that for this initial adjustment PR.

Feel free to give it a test!

**Comment by alexr4339** [comment]
Quality Assurance Tester/Trainee Report

Discord Username  : alexdnzdmb 
Object of testing : #9358  
Aircraft          : A380X 
Tier of Testing   : 1
Date              : 25/12/2024 

Testing Process: 
Takeoff and compare estimates/actual and simbrief

Flight 1:
509T (41.5%) ZFW 334T (36.6%)
Simbrief | Pred | Act
DF149  3400 | 4400
DF151  6100 | 6700
DF167 ---- | 11600
DF169  13400 | 13100
AGOLO  18500 | 19100
OKTUM 21700 | 22800
KOMIB 24200 | 25600
SULUS 27400 | 29300
TONSU 30000 | 32300

Testing Results: 
WIP

Negatives:       


Conclusions: 


Media:

**Comment by mattking247** [comment]
Quality Assurance Tester Trainee Report

Discord Username : Matt
Object of testing : https://github.com/flybywiresim/aircraft/pull/9358
Aircraft : A380X
Tier of Testing : 1
Date : 26/12/2024

Testing Process:
Full flight OMDB - EGLL, full departure from OMDB Rwy 30R with calm weather.

Notes:
505 tonnes at take off.

Various takeoffs and climbs at light and heavy weights. 

I think this feels as much like an a380 now as I can tell, without jumping in an FMS and doing a direct comparison. I think it is close enough that it feels right. Acknowledged about the THR vs N1 discussion and derates. However, as an interim before they are addressed, I think it feels really good.

I would say this passed.

**Comment by aadee9940** [comment]
Currently doing a flight from VHHH TO OMDB, and the engines are running at a constant 95% thr just to maintain airspeed. Any attempt to step climb does not work. 
image

Climbing from FL380 TO 400 at 350 tons is proving to be an impossible task. 

Because of the engines running at a constant 95% THR, fuel consumption is also greatly increased, and it looks like I am not going to make it to Dubai. :( 

Never had this behavior before though, from the few flights i have done, the engines run at around 80-85% THR. 

This is one flight though, it may not occur with others.. 
Thanks.
